samples/client/petstore/powershell_test/src/IO.Swagger/API/PetApi.ps1

```diff
@@ -0,0 +1,164 @@
+function Invoke-PetApiAddPet {
+    [CmdletBinding()]
+    Param (
+        [Parameter(Position = 1, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true, Mandatory = $true)]
```

Comment:
Position should start from `0`. Unfortunately, I couldn't figure out how to do this with `jmustache`.